internal/controller/controllers/agentserviceconfig_controller_test.go

```diff
@@ -320,6 +277,115 @@ var _ = Describe("ensureAssistedServiceDeployment", func() {
 
 })
 
+var _ = Describe("newAssistedServiceDeployment", func() {
```

Comment:
I'm being picky now, but if we have a unit test with only one of two parameters and with zero out of two, there deserves to be also test setting them both and checking if both volumes are mounted :)

Reply:
Made the change to require the registries.conf key so we are now testing: registries.conf, registries.conf + ca-bundle, and not specifying registries.conf. Let me know if I missed a scenario.

Reply:
Nice, thanks!